ruff.toml

```diff
@@ -0,0 +1,36 @@
+required-version = ">=0.11.2"
+line-length = 100
+indent-width = 4
+target-version = "py312"
+respect-gitignore = true
+
+include = [
+    "tests/test_compute_*.py", 
+    "eyepop/compute/*", 
+]
```

Comment:
trying not to overwhelm the linter... can remove once more has been chipped away at.